.gitmodules

```diff
@@ -10,3 +10,12 @@
 [submodule "lib/mbedtls"]
 	path = lib/mbedtls
 	url = https://github.com/ARMmbed/mbedtls
+[submodule "lib/mbed-cloud-client"]
+	path = lib/mbed-cloud-client
+	url = https://github.com/PelionIoT/mbed-cloud-client
```

Comment:
@prakharbhatt007 mbed-cloud-client is not a submodule as of now. Should we also convert that to submodule?

Reply:
yes.. I think we should convert that to submodule.